content/en-US/api/fontVariantCaps.md

```diff
@@ -0,0 +1,101 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontVariantCaps'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontVariantCaps
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontVariantCaps
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps).**
+
+## Description
+
+The `fontVariantCaps` property of `CanvasRenderingContext2D` allows developers
+to control the use of alternate glyphs for capital letters. If multiple sizes 
+of capital letter glyphs are available for the chosen font, this 
+`CanvasRenderingContext2D` property will choose the one with appropriate size. 
+Otherwise it synthesizes small-caps by using uppercase glyphs. The default 
```

Comment:
I think we could come up with a more descriptive verb. than 'using'. 'Adapting'? 'Altering'? I prefer 'adapting'.